src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/notifications/ReferralNotificationService.kt

```diff
@@ -0,0 +1,101 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service.notifications
```

Comment:
_suggestion_
Would it make sense to put all instances of `ApplicationListener` to a new `listeners` package? (So it's simpler to track event flow)